companies/doctolib.json

```diff
@@ -0,0 +1,30 @@
+{
+    "slug": "doctolib",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "DOCTOLIB GMBH",
+    "runs": [
+        "Doctolib SAS"
+    ],
+    "address": "Mehringdamm 51\n10961 Berlin\nDeutschland",
+    "phone": "+49 89 20702884",
+    "email": "datenschutz@doctolib.de",
```

Comment:
As this email is privacy-related you can set "suggested-transport-medium" to "email"